packages/babel-plugin-transform-es2015-parameters/src/default.js

```diff
@@ -11,6 +11,12 @@ const buildDefaultParam = template(`
       DEFAULT_VALUE;
 `);
 
+const buildLooseDefaultParam = template(`
+  if (VARIABLE_NAME === undefined) {
```

Comment:
Yeah, should change that to be a variable and then pass `scope.buildUndefinedNode()`